src/reim/autoPhaseCorrection.js

```diff
@@ -7,23 +7,19 @@ import { phaseCorrection } from './phaseCorrection';
  */
```

Comment:
Could you add the documentation for the options ?

Reply:
It's done